administrator/components/com_associations/views/associations/tmpl/default.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_associations
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+JHtml::addIncludePath(JPATH_COMPONENT . '/helpers/html');
+
+JHtml::_('bootstrap.tooltip');
+JHtml::_('behavior.multiselect');
+JHtml::_('formbehavior.chosen', 'select');
+JHtml::_('jquery.framework');
+
+$app    = JFactory::getApplication();
+$user   = JFactory::getUser();
+$userId = $user->get('id');
+$listOrder = $this->escape($this->state->get('list.ordering'));
+$listDirn  = $this->escape($this->state->get('list.direction'));
+
+$input = $app->input;
+
+$assoc = JLanguageAssociations::isEnabled();
+
+$componentFilter = $this->state->get('associationcomponent');
+
+if ($componentFilter != '')
+{
+	$parts     = explode('.', $componentFilter);
+	$comp      = $parts[0];
+	$assocItem = $parts[1];
+
+	JHtml::addIncludePath(JPATH_ADMINISTRATOR . '/components/' . $comp . '/helpers/html');
+
+	// Get the value in the Association column
+	if ($comp == "com_content")
+	{
+		$assocValue = "contentadministrator.association";
+	}
+	elseif ($comp == "com_categories")
+	{
+		$assocValue = "categoriesadministrator.association";
+	}
+	elseif ($comp == "com_menus")
+	{
+		$assocValue = "MenusHtml.Menus.association";
+	}
+	else
+	{
+		$assocValue = $assocItem . '.association';
+	}
+
+	// If it's not a category
+	if ($componentFilter != '' && !strpos($componentFilter, '|'))
+	{
+		$componentSplit = explode('.', $componentFilter);
+		$aComponent = $componentSplit[0];
+		$aView = $componentSplit[1];
+	}
+	elseif ($componentFilter != '') {
+		$componentSplit = explode('|', $componentFilter);
+		$aComponent = 'com_categories';
+		$aView = $componentSplit[1];
+	}
+
+	if (isset($aComponent) && isset($aView))
+	{
+		$link = 'index.php?option=com_associations&view=association&layout=edit&acomponent='
+		. $aComponent . '&aview=' . $aView . '&id=';
+	}
+}
+?>
+
+<?php if ($assoc) : ?>
```

Comment:
please remove assoc here.
This check needs to be done in the view html file as described in https://github.com/joomla-projects/gsoc16_improved-multi-lingual/issues/6